js/containers/compare-answer.js

```diff
@@ -18,7 +18,7 @@ export class CompareAnswerRmLink extends Component {
   render() {
     const { answer, onChange, children } = this.props
     return (
-      <a onClick={(e) => onChange(answer.get('key'), false)}>{children}</a>
+      <a onClick={(e) => onChange(answer.get('key'), false)}>{responses}</a>
```

Comment:
I think this was changed by accident. `this.props.children` refers to children DOM elements or components.